src/main/webapp/WEB-INF/views/driver/createDriver.jsp

```diff
@@ -0,0 +1,25 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
```

Comment:
let's make all names for your jsps consistent
you have now `createDriver.jsp` and `addNewCar.jsp`